web/tools/webpack.config.ts

```diff
@@ -124,14 +123,10 @@ const createConfig = (
   // If version_name is not supplied read it from version file
   const versionName = passedVersionName || readVersionName()
   const shortCommitSha = passedCommitSha?.substring(0, SHORT_COMMIT_SHA_LENGTH) || 'Commit SHA unknown'
-  // eslint-disable-next-line no-console
```

Comment:
Why are a lot of these not necessary anymore?

Reply:
tools has different linting rules. See eslint overrides: 
```ts
{
     files: ['**/tools/**', 'translations/**', 'e2e-tests/**'],
      rules: {
        'no-console': 'off',
        'import/no-extraneous-dependencies': 'off',
     }
}
```

Reply:
Ahh nvm, stupid question. Sorry.